overlord/snapstate/snapstate.go

```diff
@@ -1909,6 +1914,16 @@ func Remove(st *state.State, name string, revision snap.Revision, flags *RemoveF
 		if tp, _ := snapst.Type(); tp == snap.TypeApp && removeAll {
 			ts, err := AutomaticSnapshot(st, name)
 			if err == nil {
+				sz, err := EstimateSnapshotSize(st, name, nil)
```

Comment:
sorry, my previous comment maybe was unclear, I was more referring to the 10% than the 1mb , maybe always keeping a fixed margin is a good idea, not sure whether it should be 1mb or more though

Reply:
@mvo5 ideas on this?

Reply:
For big snapshots (think lxd) 10% can be a lot, so I think removing that is good, a fixed margin sounds good to me too, just for safety. Maybe 5Mb? if diskspace is so low that 5Mb is a problem i think we are already in the "low-diskspace" area (we can always tune it or make it configurable).

Reply:
Makes sense, thank you. Done.